src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3747,6 +3780,21 @@ Activate by holding the key for the character you want to add an accent to, then
   <data name="GeneralPage_EnableExperimentation.Header" xml:space="preserve">
     <value>Allow experimentation with new features</value>
   </data>
+  <data name="General_DiagnosticsAndFeedback.Header" xml:space="preserve">
+    <value>Diagnostics &amp; feedback</value>
+  </data>
+  <data name="GeneralPage_DiagnosticsAndFeedback_Link.Content" xml:space="preserve">
+    <value>Learn more about the information we log &amp; how it gets used</value>
+  </data>
+  <data name="DataDiagnostics_SettingIsGPOManaged_InfoBar.Title" xml:space="preserve">
+    <value>PowerToys diagnostic data is disabled because your organization has blocked permissions to change this setting.</value>
+  </data>
```

Comment:
Not required. There is a standard string we use on all single settings that are gpo managed.